src/main/java/pages/ContactUsPage.java

```diff
@@ -24,4 +32,45 @@ public Boolean isTitleVisible(){
         return driver.findElement(contactUsPageTitle).isDisplayed();
     }
 
+    public Boolean isFirstnameVisible(){
+        WebDriverWait wait = new WebDriverWait(driver,timeOut);
+        wait.until(ExpectedConditions.presenceOfElementLocated(firstName));
+        return driver.findElement(firstName).isDisplayed();
+    }
+
+    public Boolean isLastnameVisible(){
+        WebDriverWait wait = new WebDriverWait(driver,timeOut);
+        wait.until(ExpectedConditions.presenceOfElementLocated(lastName));
+        return driver.findElement(lastName).isDisplayed();
+    }
+
+    public Boolean isCompanyVisible(){
+        WebDriverWait wait = new WebDriverWait(driver,timeOut);
+        wait.until(ExpectedConditions.presenceOfElementLocated(companyName));
+        return driver.findElement(companyName).isDisplayed();
+    }
+
+    public Boolean isPhoneVisible(){
+        WebDriverWait wait = new WebDriverWait(driver,timeOut);
+        wait.until(ExpectedConditions.presenceOfElementLocated(phone));
+        return driver.findElement(phone).isDisplayed();
+    }
+
+    public Boolean isEmailVisible(){
+        WebDriverWait wait = new WebDriverWait(driver,timeOut);
+        wait.until(ExpectedConditions.presenceOfElementLocated(email));
+        return driver.findElement(email).isDisplayed();
+    }
+
+    public Boolean isMessageVisible(){
+        WebDriverWait wait = new WebDriverWait(driver,timeOut);
+        wait.until(ExpectedConditions.presenceOfElementLocated(message));
+        return driver.findElement(message).isDisplayed();
+    }
+    public Boolean isSendVisible(){
+        WebDriverWait wait = new WebDriverWait(driver,timeOut);
+        wait.until(ExpectedConditions.presenceOfElementLocated(sendButton));
+        return driver.findElement(sendButton).isDisplayed();
```

Comment:
Lots of code duplication in the above six methods. Can you find a good way to reduce the duplication and have the code abide the SOLID principles better?